docs/running.rst

```diff
@@ -33,8 +33,48 @@ To setup the server locally:
     $ mkdir ddb
     $ curl -sSL http://dynamodb-local.s3-website-us-west-2.amazonaws.com/dynamodb_local_latest.tar.gz | tar xzvC ddb/
     $ java -Djava.library.path=./ddb/DynamoDBLocal_lib -jar ./ddb/DynamoDBLocal.jar -sharedDb -inMemory
-    $ cp automock/boto.cfg ~/.boto
 
-Note the last line copies a boto config over ``~/.boto`` in your home dir. If
-you have existing AWS Credentials in this file, you should move it elsewhere
-first before running ``autopush`` and ``autoendpoint``.
+An example `boto config file`_ is provided in ``automock/boto.cfg`` that
```

Comment:
Huh, wonder if we should copy this to ./configs, since that's where we've collected other, similar config files. Probably ought to comment the file as well. (Not that it's helped in some cases, but it's consistent)